plaso/parsers/manager.py

```diff
@@ -351,35 +243,42 @@ def GetParserObjects(cls, parser_filter_expression=None):
     return parser_objects
 
   @classmethod
-  def GetParsers(cls, parser_filter_expression=None):
+  def _GetParsers(cls, parser_filter_expression=None):
     """Retrieves the registered parsers and plugins.
 
     Retrieves a dictionary of all registered parsers and associated plugins
     from a parser filter string. The filter string can contain direct names of
-    parsers, presets or plugins. The filter string can also negate selection
-    if prepended with an exclamation point, e.g.: "foo,!foo/bar" would include
-    parser foo but not include plugin bar. A list of specific included and
-    excluded plugins is also passed to each parser's class.
+    parsers or plugins. The filter string can also negate selection if prepended
+    with an exclamation point, e.g.: "foo,!foo/bar" would include parser foo but
+    not include plugin bar. A list of specific included and excluded plugins is
+    also passed to each parser's class.
 
     The three types of entries in the filter string:
      * name of a parser: this would be the exact name of a single parser to
        include (or exclude), e.g. foo;
-     * name of a preset, e.g. win7: the presets are defined in
-       plaso/parsers/presets.py;
      * name of a plugin: if a plugin name is included the parent parser will be
        included in the list of registered parsers;
 
     Args:
-      parser_filter_expression (Optional[str]): parser filter expression,
+      parser_filter_expression (Optional[str]): the parser filter expression,
```

Comment:
Why "the" here? There's no specific filter expression required here, any expression is valid.

Reply:
because this is used elsewhere as well